helm-charts/basehub/values.yaml

```
@@ -449,11 +449,7 @@ jupyterhub:
           - |
             ls -lhd /srv/repo;
             while true; do git fetch origin;
-            if [[ $(git ls-remote --heads origin $(HUB_TEMPLATES_REPO_BRANCH) | wc -c) -ne 0 ]]; then
-                git reset --hard origin/$(HUB_TEMPLATES_REPO_BRANCH);
-            else
-                git reset --hard origin/master;
-            fi
+            git reset --hard origin/$(HUB_TEMPLATES_REPO_BRANCH);
```

I think the syntax for referencing an env variable to use here, is valid if you were in the k8s `env` section defining another env var based on another env var.

This is the container command, inside a `sh` based shell though, so I think the syntax should be `$SOMETHING` or `${SOMETHING}` instead.

```suggestion
            git reset --hard origin/${HUB_TEMPLATES_REPO_BRANCH};
```